### ISSUE TITLE
Add option for `anomaw address find` to output JSON

### ISSUE THREAD

**Issue by james-chf** [anoma/anoma#1157]
Currently in anoma v0.6.0, this command prints a human-friendly string to stdout:

```
testrunner@176d85e46406:/opt/test$ anomaw address find --alias multitoken-established
Found address Established: atest1v4ehgw36g4pyg3j9x3qnjd3cxgmyz3fk8qcrys3hxdp5xwfnx3zyxsj9xgunxsfjg5u5xvzyzrrqtn
```

It'd be helpful for scripting and interoperability if there was also an option to output something more structured and less likely to change eg

```json
{
    "type": "Established",
    "bech32m": "atest1v4ehgw36g4pyg3j9x3qnjd3cxgmyz3fk8qcrys3hxdp5xwfnx3zyxsj9xgunxsfjg5u5xvzyzrrqtn"
}
```

Having an option to output JSON could be useful for some of our other `anomaw` (and `anomac`) commands, especially the ones we call during our e2e tests that we end up parsing and asserting on the stdout output for.

**Comment by Fraccaman** [comment]
this would be super helpful but I think we should extend the whole rpc client to with a `--json` argument

**Comment by james-chf** [comment]
Yes, ideally for every command we would have a human-friendly output and machine-friendly (`--json`) output. So this `--json` flag could be implemented as a global flag
